.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: rust
+sudo: false
+
+matrix:
+  include:
+
+  # tests pass
+  - rust: stable
+    script:
+    - cargo test --locked
+    - rustup component add rustfmt-preview
+    - cargo fmt --all -- --check
+    env: RUST_BACKTRACE=1
+
+  # dist linux binary
+  - env: TARGET=x86_64-unknown-linux-musl DEPLOY=1
+    rust: stable
+    before_script:
+      - rustup target add $TARGET
+      - curl https://www.openssl.org/source/openssl-1.0.2l.tar.gz | tar xzf -
+      - (cd openssl-1.0.2l &&
+        CC=musl-gcc ./Configure --prefix=$HOME/openssl-musl no-dso no-ssl2 no-ssl3 linux-x86_64 -fPIC &&
+        make -j$(nproc) &&
+        make install)
+      - export OPENSSL_DIR=$HOME/openssl-musl
+    script: cargo build --release --target $TARGET --locked
+
+  # dist OSX binary
+  - os: osx
+    rust: stable
+    env: MACOSX_DEPLOYMENT_TARGET=10.7 DEPLOY=1 TARGET=x86_64-apple-darwin
+    script: cargo build --release --target $TARGET --locked
+    install: true
```

Comment:
Not sure what these dist builds do aside from building the project right now? Are they still useful if we don't do deploys (cf #7)?

Reply:
oh yeah whoops. i can fix, i am just cargo culting from the wasm-pack one, which has lot more going on 😅